modules/administration-event-details-overview.adoc

```diff
@@ -3,17 +3,36 @@
 // *operating/using-the-administration-events-page.adoc
 
 :_mod-docs-content-type: CONCEPT
-[id="understanding-the-administration-event-details_{context}"]
-= Understanding the administration event details
+[id="administration-event-details-overview_{context}"]
+= Administration event details overview
 
-The administration event provide log information that describes the error or event. They provide important information for understanding a specific issue or incident and includes information about the context of the event as well as the steps to take to fix the error.
+The administration event provide log information that describes the error or event. 
+
+The logs provide the following information:
+
+* Context of the event.
```

Comment:
This is a bulleted list of incomplete sentences (fragments), so periods are not needed. See [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#punctuation-in-lists).